mkosi/__init__.py

```diff
@@ -4113,6 +4113,7 @@ def secure_boot_sign(
     for_cache: bool,
     cached: bool,
     mount: Callable[[], ContextManager[None]],
+    esp: bool,
```

Comment:
I think a more general name like `replace` or `overwrite` might be more suitable.

Reply:
Fixed.